### PR TITLE
status-updates: Add summary of meeting on Thursday, August 17, 2023

### PR DESCRIPTION
**Pull request by teodutu**

### CODE REVIEW

File: status-updates/2023/2023-08-17.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+RD: The methodology must answer the following questions:
+
+- How can I develop / create a digital educational content repository? 
+- How do I use (as an educator) a digital educational content repository as part of an educational implementation (course, training, workshop, etc.)?
+- How do I contribute to a digital educational content repository?
+- How do I maintain a digital educational content repository?
+
+RD: We are focusing on how to use the resources and are targeting the developer, teacher, contributor and manager / curator/
+Administrating the infrastructure is something unrelated.
+We need to assign roles per section so any of the above users knows where to look.
```
Comment (Alex-deVis) [inline status-updates/2023/2023-08-17.md]:
```suggestion
RD: We are focusing on how to use the resources and are targeting the developer, teacher, contributor, and manager/curator.
Administrating the infrastructure is something unrelated.
We need to assign roles per section so any of the above users know where to look.
```

File: status-updates/2023/2023-08-17.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+RD: We are proposing the following repository structure:
+
+- A content repository that is organised and used as described in the "Developing and Organizing Content" section.
```
Comment (Alex-deVis) [inline status-updates/2023/2023-08-17.md]:
```suggestion
- A content repository that is organized and used as described in the "Developing and Organizing Content" section.
```
zz

File: status-updates/2023/2023-08-17.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+In addition, this section should contain any commands or technical parts.
+
+RD: And then we'll refer the infrastructure components from the following sections as needed.
+Make "Tools" a section apart from the general infrastructure.
+
+RD: Rename "Teaching Stages" to "Using and Delivering Content"
+
+RD: Now the structure becomes: infrastructure, Developing and Organizing Content, Using and Delivering Content
+
+SW: Now it makes more sense and is more coherent.
```
Comment (Alex-deVis) [inline status-updates/2023/2023-08-17.md]:
```suggestion
SW: Now it makes more sense and it is more coherent.
```

File: status-updates/2023/2023-08-17.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+I'll raise issues for the builder.
+
+RD: Git itself will be difficult to use.
+We can think of abstractions on top of it, something that simplifies the interaction via an interface.
+
+RD: We can also provide developers with an exhaustive `.yml` file from which they will have to remove what they don't want to use.
+
+SW: I'll think about a config file generator, maybe based on some regular expressions.
+
+GM + RD: This sounds like a very solid idea.
```
Comment (Alex-deVis) [inline status-updates/2023/2023-08-17.md:156]: To be clear, `+` means that they talk like [team rocket](https://www.youtube.com/watch?v=ZBVrPWwSlRM).

File: status-updates/2023/2023-08-17.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+
+RD: And use drills for 3 use cases:
+
+- exercising: labs / seminars
+- something quick during lectures
+- self-study
+
+RD: This is also what sets projects apart from drills.
+
+RD: we'll need to merge drills and quizzes after merging the PRs.
```
Comment (Alex-deVis) [inline status-updates/2023/2023-08-17.md]:
```suggestion
RD: We'll need to merge drills and quizzes after merging the PRs.
```

File: status-updates/2023/2023-08-17.md (hunk 2 above)
Comment (teodutu) [reply status-updates/2023/2023-08-17.md]: Both are correct. It's just that my default is the British spelling and not the American one.

Comment (Alex-deVis) [reply status-updates/2023/2023-08-17.md]: Yes, I am aware, I was just pointing out your inconsistency. Pick one and reformat.

File: status-updates/2023/2023-08-17.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+The methodology is a volatile document, we'll keep modifying and improving it as time passes.
+
+RD: Right.
+After merging, I'll raise a new PR with an update of the methodology repo based on its own guidelines and then we'll add content to the new structure.
+
+### Difficulties when Porting SSS to OpenEdu
+
+GM: I had to break up the contents of the repository in a great number of folders.
+If someone external attempts to do the same, they will find it extremely cumbersome to create such a folder hierarchy from the GUI.
+So reorganising the content is likely going to be a difficult task.
```
Comment (Alex-deVis) [inline status-updates/2023/2023-08-17.md]: Last one, I'll even pre-approve this.

**Review by Alex-deVis — APPROVED** [review]

**Pull request merged**